code/style.css

```diff
@@ -2,6 +2,245 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+/*
 .active + .answer {
   /* some styling here */
+/*}*/
+
+body {
+ margin: 0;
+ font-family: 'Quicksand', Arial, Helvetica, sans-serif;
+ background: black;
+}
+.header{
+  display: flex; 
+  position: relative;
+  margin: 0;
+  padding: 0;
+}
+.hero-image {
+  width: 100vw;
+  height: 30vh;
+  object-fit: cover;
+}
+.hero-title {
+  position: absolute;
+  color: #ffff;
+  font-size: 40px; 
+  left: 20px;
+  top: 20px
+}
+.container-menu {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  color: #fff;
+  font-size: 20px;
+}
+.menu {
+  text-align: center;  
+  border-bottom: 1px solid whitesmoke;
+  width: 100%;
+  transition: background 2s ease;
+}
+.menu:hover {
+  background: linear-gradient(to left, #29323c, #485563, #2b5876);
+}
+.container-hero {
+  position: relative;
+}
+.container-forms {
+  background: white; 
+  color: black;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  border-radius: 20px;
+  margin: 20px;
+  width: 350px;
+}
+.form {
+  display: flex;
+  background: black;
+  color: white;
+  cursor: pointer;
+  }
+/* yoga level dropdown */
+input [type=text], select { 
+  padding: 5px;
+  border-radius: 5px;
+  font-size: 14px;
+  width: 250px;
+  font-family: 'Quicksand', Arial, Helvetica, sans-serif;
+}
+.yoga-level {
+  padding-bottom: 30px; 
+}
+.input-fields {
+  padding-bottom: 
+}
+/* Input fields */
+.input {
+  width: 100%;
+  margin-top: 10px;
+  border: none;
+  border-bottom: 2px solid rgb(11, 10, 10);
+  transition: border-color 0.4s ease;
+  font-size: 14px;
+  padding: 10px 0;
+}
+.input:focus {
+  outline: none;
+  border-color:#377198;
+}
+.input:valid {
+  color: rgb(12, 151, 12);
+}
+.checkbox {
+  padding-top:20px;
 }
+.submit-button {
+  background: #173252;
+  color: whitesmoke;
+  font-size: 14px;
+  box-sizing: border-box;
+  border-radius: 20px;  
+  box-shadow: 0 4px #999;
+  margin-top: 30px;
+  display: block;
+}
+.submit-button:active {
+  background-color: #29323c;
+  box-shadow: 0 1px rgb(80, 79, 79);
+  transform: translateY(5px);
+}
+.container-faq {
+  display: flex;
+  border-radius: 20px;
+  padding:10px;
+}
+.accordion-faq {
+  font-size: 20px;
+  font-weight: 600;
+  padding: 20px;
+  width: 50%;
+  margin: 20px;
+  font-size: 20px;
+}
+.accordion {
+  margin: 0px;
+  display: flex;
+  flex-direction: column;
+  color: black;
+  border-radius: 5px;
+  cursor: pointer;
+  width: 300px;
+  border:2px solid #2b5876; 
+}
+.accordion-question:hover {
+  background: linear-gradient(to left, #29323c, #485563, #2b5876);
+}
+.accordion-question {
+  margin: 0;
+  padding: 10px;
+  font-weight: 200;
+  border-radius: 5px;
+  font-size: 19px;
+  border: 2px solid black;
+  transition: background 2s ease-in-out;  
```

Comment:
Really like this styling! Not sure when this transition is supposed to happen though 😇